src/components/containers/show.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { useContext, ReactNode } from 'react'
+import { Media } from 'themes/media'
+import { DerivStore } from 'store'
+
+type ResponseType = {
+    children?: ReactNode
+    max_width?: string & number
+    min_width?: string & number
+}
+
+export const Desktop = ({ children, max_width, ...props }: ResponseType) => (
+    <Media greaterThanOrEqual={max_width || 'tabletL'} {...props}>
+        {children}
+    </Media>
+)
+
+// We can't type it properly due fresnel
```

Comment:
is this comment needed?

Reply:
Seems we should explain why we decide to use any here